### ISSUE TITLE
Biggest Issue of Kivy Platform That's Depends on OpenGL By Forced 

### ISSUE THREAD

**Issue by tejastank** [kivy/kivy#5251]
* Python: 2.7
* OS: Microsoft OS 10.0, Microsoft 2012 server
* Kivy: 1.9 & lastest
* Kivy installation method: via PIP 

We build solution for clients after 2 month of development we run with remote desktop (RDP)
Kivy not supported with RDP gives OpenGL 1.1 only, this make biggest failer over the development.

Now complete kivy project failuer due to only it unable to run on microsft server 2012.
Client conenct with RDP and just all get OpenGL 1.1 Issue.

Even we upgrade costly graphics card in server with OpenGL 4.4 supported. 
Application directly work in server with python and we generated exectutable both.

But ALL RDP or AnyDesk App Failed to run it. RDP must need to run.

Is their any way to remove OpenGL from kivy Application ?

Let me know if any screenshot required I can provide if some one from Kivy Official Expert Able to resolve it.

**Comment by dessant** [comment]
Windows Server 2012 does not support OpenGL 2 over RDP, but Windows Server 2016 does.

You could also search for a third-party RDP software which supports OpenGL 2.

**Comment by matham** [comment]
I believe it'll work with angle on python 3.5+, if you `pip install kivy.deps.angle` and then set in the environment `set KIVY_GL_BACKEND=angle_sdl2`. But you also need to change your appveyor script to have this:

```
install:
- ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-desktop.ps1'))
```.

On python < 3.5 you can install the same kivy.deps.angle wheel file as for 3.5 by changing the wheel filename from 3.5 to 2.7, the one from pypi for 2.7 probably will not work.

However, as @dessant said, you need 2016 if you want to support a graphics card without angle.

**Comment by tejastank** [comment]
We upgraded OpenGL 4.4 in Window 2012 server with additional hardware and all works fine now.

+ working well with remote desktop / any desk too.

Video with kivy app + remote desk / anydesk working well.
https://youtu.be/B8_9DC9mjkY

**Comment by tejastank** [comment]
@matham 

Explain more on this, seems nothing clear, I am ubuntu user, rar use window.

install:
- ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-desktop.ps1'))

As you experienced with Python 3++, kindly share video or documenatation to build it.

**Comment by matham** [comment]
Didn't you just say in the previous post that it is working for you after you upgraded to additional hardware?

My original post was in response to the original question of how to get it to work on MS server. On Ubuntu things are different. Please ask on the kivy users list or on IRC at #kivy because github is meant for bug reports, not support.

**Comment by tejastank** [comment]
@matham bug report at X and support at Y as defined in your brain, not invidual on earth may don't know.

# Seems your comment are not productive, In case you had experties on Kivy, please answer below.

-> Now issue happen when I connected window 2012 from other system via remote desk to start kivy executbale it gives OpenGL 1.1 issue.

-> Case when we start Kivy Executable from Window2012 than connect via remoted desktop than it works.

So only issue happen only unable to run it from remote desktop, what should we do ?

**Comment by dessant** [comment]
@tejastank, you have been banned from interacting with the Kivy project for 30 days. We welcome you back after you adjust your attitude towards people who spend time helping you.

**Comment by tejastank** [comment]
@dessant its very rude from you. I had not derespect anyone... 

All great thanks to all people who answers.

Here don’t get any rock solids answer with official kivy framework.

OpenGL related issues always generate, their no really right fix which 1000% works.

Even you are moderator, you had authority to rudely ban anyone...but not resolve issues.

@matham and all great thanks for your all effort and communication.

**Comment by Spartacus1066** [comment]
@dessant you should know the difference between being rude and a language barrier. I think he just had issues finding the right words to express the issue as you would think in a polite matter.

**Comment by dessant** [comment]
@Spartacus1066, at the time we had an internal discussion and we have concluded that despite the user being redirected to the appropriate support channel, they continued to insist on getting support here, so their access was temporarily restricted. I think none of use called the user rude.

**Comment by tejastank** [comment]
Since "on 30 Jun 2017" As I reported, I am awaiting for fix, so we / world can reuse kivy framework over given situation. Looking for positive waves.

**Comment by dessant** [comment]
@tejastank, your question does not appear to be a bug report, but a support request, so it's best to join https://chat.kivy.org and ask your question on the #support channel.

**Comment by tshirtman** [comment]
You'll be waiting a long time, as this has been closed, kivy does require opengl to work, maybe you can have it working with the angle provider, which already exists, so either it's possible already, and no code fix is required, or it's not possible, and we don't intent to try, so no fix will be provided, as it's out of our scope. If you need help with angle, go to support, and if your issue is now different than originaly stated, go to support anyway to ask for help, we'll tell you if it's worth opening a bug. Best regards.

**Comment by matham** [comment]
I've locked the issue. I noticed you edited your original comment that caused the ban to make it more polite than what it originally was. 

So it's great to see you now understand that your original comments, directed at the people who were trying to help you, are inappropriate. However, you are still asking for support for an issue that we made clear we don't consider a bug and that this is not the right place to as for such support. Hence I'm locking the issue.
